### ISSUE TITLE
Keyboard shortcut "v" in label studio gives error

### ISSUE THREAD

**Issue by hummuscience** [Lightning-Universe/Pose-app#124]
Weird behavior. I haven't found the reason for it but I guess label studio avoids certain keyboart shortcuts when assigning them to labels for that reason.

I had many labels (27) and happened to have one of them on the shortcut "v". Every time I would press it, it would stop me from labelling the next point. But clicking the label with the mouse worked fine.

I changed the hotkey to "m" like so in the label studio settings:     `<Label value="right_shoulder" hotkey="m" />`

That solved it.

Not sure it if makes sense avoid this somewhere

**Comment by hummuscience** [comment]
I see. pressing "v" turns on "move" and "h" is for "hand".

**Comment by hummuscience** [comment]
Will try out something like this to add color and hotkey directly to the labels (the colors were also too close to each other by default).
```

def build_xml(bodypart_names):
    """Builds the XML file for Label Studio"""
    # 25 colors
    colors = ["red", "blue", "green", "yellow", "orange", "purple", "brown", "pink", "gray",
              "black", "white", "cyan", "magenta", "lime", "maroon", "olive", "navy", "teal",
              "aqua", "fuchsia", "silver", "gold", "indigo", "violet", "coral"]
    hotkeys = ["1","2","3","4","5","6","7","8","9","0",
               "q","w","e","r","t","y","u","i","o","p",
               "a","s","d","f","g","j","k","l",
               "z","x","c","v","b","n","m"]
    # replicate just to be safe
    colors = colors + colors + colors + colors
    hotkeys = hotkeys + hotkeys
    colors_to_use = colors[:len(bodypart_names)]  # practically ignoring colors
    hotkeys_to_use = hotkeys[:len(bodypart_names)]
    view_str = "<!--Basic keypoint image labeling configuration for multiple regions-->"
    view_str += "\n<View>"
    view_str += "\n<Header value=\"Select keypoint name with the cursor/number button, " \
                "then click on the image.\"/>"
    view_str += "\n<Text name=\"text1\" value=\"Save annotations: " \
                "click Submit (or CTRL+ENTER)\"/>"
    view_str += "\n<Text name=\"text2\" value=\"Manipulate image: press H for hand tool, " \
                "\nCTRL+ to zoom in and CTRL- to zoom out\"/>"
    view_str += "\n<Text name=\"text3\" value=\"Next frame: SHIFT+DOWN, then SHIFT+RIGHT\"/>"
    view_str += "\n<Text name=\"text4\" value=\"To copy keypoints to another frame: hold CTRL " \
                "and select all keypoints; CTRL+c to copy; move to new frame; CTRL+v to paste\"/>"
    view_str += "\n  <KeyPointLabels name=\"kp-1\" toName=\"img-1\" strokeWidth=\"3\">"  # indent 2
    for keypoint, color, hotkey in zip(bodypart_names, colors_to_use,hotkeys_to_use):
        view_str += f"\n    <Label value=\"{keypoint}\" background=\"{color}\" hotkey=\"{hotkey}\" />"  # indent 4
    view_str += "\n  </KeyPointLabels>"  # indent 2
    view_str += "\n  <Image name=\"img-1\" value=\"$img\" />"  # indent 2
    view_str += "\n</View>"  # indent 0
    return view_str
```

**Comment by hummuscience** [comment]
Yeah. It will break things.

The best is probably to make a list of the hotleys to be avoided (h,v, any others?) And then substact those from a list of hotkeys (with the assumption that people are using US keyboard, which doesn't apply to my institute)

Label Studio seems to already do that somehow. Maybe it's just a matter of adding a few more to the "avoided" list.

**Comment by themattinthehatt** [comment]
I checked all of the keys and only `h` and `v` are mapped to other functions, so no worries there.

I also checked what happens when you set `hotkeys = hotkeys + hotkeys` and you have more than `len(hotkeys)` keypoints. The good news is that labelstudio fails pretty gracefully - the double-mapped hotkey just defaults the the latest assigned keypoint, rather than breaking something. So I've left that as-is for now, and if someone has more than 34 keypoints then they'll just have to manually click some of the keypoints rather than use hotkeys for now.
